wetterdienst/provider/dwd/observation/metadata/parameter.py

```diff
@@ -845,14 +845,14 @@ class WEATHER_PHENOMENA(Enum):  # noqa
         # DwdObservationParameter.MONTHLY.PRECIPITATION_HEIGHT: DwdObservationParameterSet.CLIMATE_SUMMARY,
```

Comment:
Remove unnecessary comments.

Reply:
I believe those comments are not unnecessary because it is a hazzle to keep an overview of the existing datasets especially for DWD. The commented lines are just a result of my personal selection - here for the mapping of parameter to dataset - but should not be removed as they represent the entirety of parameters available within all datasets. So in the case of 

```python
         # DwdObservationParameter.MONTHLY.PRECIPITATION_HEIGHT: DwdObservationParameterSet.CLIMATE_SUMMARY,
```

we are still able to recognize that the parameter `PRECIPITATION_HEIGHT` is also available in `CLIMATE_SUMMARY` dataset but we have specifically chosen not to consider it but rather use the one from `PRECIPITATION_MORE`.